indico/MaKaC/webinterface/tpls/ConfReviewingAssignContributions.tpl

```diff
@@ -8,7 +8,7 @@
 % if not ConfReview.hasReviewing():
 <p style="padding-left: 25px; color:gray;">${ _("Type of reviewing has not yet been chosen. You can choose it from Paper Reviewing ")}<a href="${urlHandlers.UHConfModifReviewingPaperSetup.getURL(ConfReview.getConference())}">${ _("Setup.")}</a></p>
 % else:
-% if len(Conference.getContributionListSorted()) == 0:
+% if not event.contributions:
```

Comment:
that queries the whole list of contributions :/ Maybe an `Event.contribution_count` column_property that's deferred by default wouldn't be a bad idea